### DISCUSSION TITLE
[Feature] Add an explore filter on images/videos without geodata

### DISCUSSION THREAD

**Discussion by freak12techno** [immich-app/immich#5536] [Feature Request]
### The feature

Let's say I want all of my images to have a geolocation.
Given that with the latest releases it's now possible to manually set a geolocation for any image/photo, I can manually set it, but I do not have a way to get a list of all my photos/videos that do not have geolocation set.
I suggest adding an "Explore" tab that would display only such images/videos, is it posslble/worthwhile adding?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This can now be done by searching for country "Unknown".
